pkg/collector/client.go

```diff
@@ -65,14 +67,25 @@ func (c *clientCollector) Collect(ch chan<- prometheus.Metric) {
 	for _, item := range clients {
 		vlanId := fmt.Sprintf("%.0f", item.VlanId)
 		port := fmt.Sprintf("%.0f", item.Port)
+		cHostName := item.HostName
+		if cHostName == "" {
```

Comment:
As mentioned in your comment, I think we should just add hostname/name as separate labels, just to give more options to the user

I'm not even sure how Omada populates this `Name` property, for my clients it's sometimes the hostname, sometimes the MAC address 🤔